Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalModule.cc

```diff
@@ -46,10 +46,10 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
     edm::LogVerbatim("HGCalGeom") << "Volume [" << i << "] " << names[i] << " of thickness " << thick[i]
                                   << " filled with " << materials[i] << " first copy number " << copyNumber[i];
 #endif
-  const auto& layers = args.value<std::vector<int> >("Layers");             // Number of layers in a section
-  const auto& layerThick = args.value<std::vector<double> >("LayerThick");  // Thickness of each section
-  const auto& layerType = args.value<std::vector<int> >("LayerType");       // Type of the layer
-  const auto& layerSense = args.value<std::vector<int> >("LayerSense");     // Content of a layer (sensitive?)
+  const auto& layers = args.value<std::vector<int> >("Layers");          // Number of layers in a section
+  auto layerThick = args.value<std::vector<double> >("LayerThick");      // Thickness of each section
```

Comment:
why is this now a copy rather than const ref? It doesn't seem like the values are changed anywhere later.